src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -166,12 +167,18 @@ trait SharedFileSystem {
   /**
    * Return a possibly altered copy of inputs reflecting any localization of input file paths that might have
    * been performed for this `Backend` implementation.
-   */
-  def adjustInputPaths(callKey: CallKey, inputs: CallInputs, workflowDescriptor: WorkflowDescriptor): CallInputs = {
+    *
+    *     * NOTE: This ends up being a backdoor implementation of Backend.adjustInputPaths as both LocalBackend and SgeBackend
```

Comment:
Never seen a note prefixed with ` * NOTE: `. Just checking if this was an intentional style preference and not a copy / paste issue?